lib/ui/form/widget/form_survey_answer_text_field.dart

```diff
@@ -0,0 +1,59 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_survey/model/question_model.dart';
+import 'package:flutter_survey/theme/app_dimensions.dart';
+import 'package:flutter_survey/ui/widget/custom_text_field.dart';
+import 'package:collection/collection.dart';
+
+class FormSurveyAnswerTextField extends StatefulWidget {
+  final QuestionModel question;
+  const FormSurveyAnswerTextField({super.key, required this.question});
+
+  @override
+  State<FormSurveyAnswerTextField> createState() =>
+      _FormSurveyAnswerTextFieldState();
+}
+
+class _FormSurveyAnswerTextFieldState extends State<FormSurveyAnswerTextField> {
+  final List<TextEditingController> _controllers = [];
+  late List<String> _textFieldHints = [];
+
+  @override
+  Widget build(BuildContext context) {
+    _textFieldHints =
+        widget.question.answers.map((element) => element.text).toList();
+    for (int i = 0; i < _textFieldHints.length; i++) {
+      _controllers.add(TextEditingController());
+    }
+    return SingleChildScrollView(
+      child: Column(
+        mainAxisAlignment: MainAxisAlignment.center,
+        children: [
+          for (var textAndControllerPair in IterableZip([
+            _textFieldHints,
+            _controllers,
+          ]))
+            Padding(
+              padding: const EdgeInsets.only(bottom: AppDimensions.spacing20),
+              child: customTextField(
+                context: context,
+                controller: textAndControllerPair[1] as TextEditingController,
```

Comment:
@luongvo Can you think of an approach that doesn't need casting? 💭 

The only approach is to create a new class for it, I miss Kotlin 🙈